test/test_open.py

```diff
@@ -12,6 +13,23 @@
 FILENAME = Path('one')
 
 
+# Windows OS helper for checking admin privileges
+def is_windows_admin():
+    """Check if the script is running in a terminal with admin privileges on Windows"""
+    if os.name == 'nt':
+        try:
+            return ctypes.windll.shell32.IsUserAnAdmin()
+        except Exception:
+            return False
+
+
+IS_WINDOWS_USER = os.name == 'nt' and not is_windows_admin()
+skip_windows = unittest.skipIf(
```

Comment:
OK, I now think that my name `skip_windows` was inferior because it's not what's really going on. 🤡 

It may be a bit long, but `skip_if_symlink_creation_forbidden` is very clear.